yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -418,7 +419,7 @@ module: ietf-yang-status-conformance
         </section>
       </section>
       <section anchor="guidelines_clients" title="Versioning Considerations for Clients">
-        <t>Guidelines for clients of modules using the new module revision update procedure:
+        <t>Guidelines for clients, and therefore also for developers of those clients, of modules using the new module revision update procedure:
```

Comment:
Maybe just:

> Guidelines for client and developers of modules using...

Reply:
I think Jurgen's comments were that the guidelines are for the developers of the clients, not the clients.

Reply:
I'm changing this to `Guidelines for developers of clients of modules using ....`